src/ui/controls/shortcut/index.tsx

```diff
@@ -88,11 +89,16 @@ export default function ShortcutLink(props: ShortcutLinkProps) {
         });
     }
 
+    function onUpdate(props) {
+        props.textContent = shortcutMessage;
+    }
+
     return (
         <a
             class={cls}
             href="#"
             onclick={onClick}
+            updated={onUpdate}
```

Comment:
Let's add `attached={onRender}`. `attached` handler is invoked after DOM element is created and attached to DOM for the first time, `updated` is called after DOM was synchronized with virtual DOM. I had an idea to add `rendered` handler that would be called in both cases.